test/test_from_sdist.py

```diff
@@ -53,6 +53,7 @@ def cibuildwheel_from_sdist_run(sdist_path, add_env=None, config_file=None):
 # tests
 
 
+@utils.skip_if_pyodide(reason="TODO: fix!")
```

Comment:
Does this just need to be updated to 312? Also a few others below.

Reply:
I got the obvious ones, there may be more tests that would work if they were against 312 but I'd say we should deal with that in followup work.